en/docs/tutorials/scenarios/scenario3-implementing-an-api.md

```diff
@@ -1,4 +1,63 @@
+
+**_Time to Complete : ? mins_**
+
+There are three different sample backends implemented which provide the metrics in different formats. When you invoke them directly, you can view metrics related to one Telecom company. 
+
+```
+curl -X GET 'http://localhost:8082/telecom-rest-service/nexus/v1/metrics' 
+
+```
+
+With Micro Integrator and Integration Studio, we can implement the business logic to call these three backends, aggregate the response and present it to the client as one response.
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/scenario_implement_api.png" alt="Implement API" title="Implement API" width="60%" />
+
+To develop a service in Micro Integrator, you need to use Integration Studio. While developing you can try it out in the Embedded Micro Integrator inside the Studio. Once the development is complete you can export it as a Compose Application and add it to the Micro Integrator runtime.
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/integration_studio_view.png" alt="Integration Studio View" title="Integration Studio View" width="60%" />
+
+Here, for simplicity, the service is already created and exported as a Composite Application and added to the Micro Integrator instance in the tutorial setup. You can test the Micro Integrator service by invoking it directly.
+
+```
+curl -X GET 'http://localhost:8290/metrics' 
+```
+
+If the service catalog is enabled in Micro Integrator, it will automatically push the API artifacts, Swagger definition to the API Manager during startup, which you can view in the API Manager Service Catalog.
+
+To enable the service catalog, uncomment the following section in the Dockerfile found inside `<REPO_HOME>/dockerfiles/micro-integrator/`.
+
+`COPY deployment.toml ${WSO2_SERVER_HOME}/conf/`
+
+This will add the new config file to setup. To apply the config, restart the Micro Integrator by running the following command. This will restart the Micro Integrator container.
+
+_`docker-compose up -d --build mi-runtime`_
+
+Once started, you can observe the following log in Micro Integrator.
+
+`{ServiceCatalogUtils} - Successfully updated the service catalog`
+
+You can view the API entry in the API manager by visiting the following URL. (Login as jill@railco.com)
+
+[https://localhost:9443/publisher/service-catalog](https://localhost:9443/publisher/service-catalog)
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/service_catalog_view.png" alt="Service Catalog View" title="Service Catalog View" width="60%" />
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/service_catalog_detail_view.png" alt="Service Detailed View" title="Service Detailed View" width="60%" />
+
+In this setup, the API is already deployed from the publisher and you can view in the DevPortal. 
+
+1. Go to [https://localhost:9443/devportal/](https://localhost:9443/devportal/) Dev portal and select **RailCo** tenant domain. This will redirect you to RailCo’s developer portal.
```

Comment:
Let's consistently use "Developer Portal" everywhere for consistency.

Reply:
Fixed